streisand

```diff
@@ -8,6 +8,7 @@ echo -e "\n\033[38;5;255m\033[48;5;234m\033[1m  S T R E I S A N D  \033[0m\n"
 
 # Update the PYTHONPATH to include the monkey patch directory for Streisand
 # workarounds packaged as Python monkey patches
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd -P)"
```

Comment:
Can you also remove [the `SCRIPT_DIR` local](https://github.com/jlund/streisand/blob/2bd2a007dc6230685b1117527037f49c2dbe606b/streisand#L38) and [the duplicate SCRIPT_DIR instantiation](https://github.com/jlund/streisand/blob/2bd2a007dc6230685b1117527037f49c2dbe606b/streisand#L40) from the `check_python` function? I agree with the fix to put this at a global scope and the duplicates can be removed.